RELEASE_NOTES

```diff
@@ -1,3 +1,15 @@
+=======================================================================
+amazon-cloudwatch-observability v1.8.0 (2024-07-02)
+========================================================================
+Bug Fixes:
+* Add GOMEMLIMIT environment variable for Neuron Monitor to fix OOM crash issue (#56)
+
+Enhancements:
+* Support Kubernetes 1.30 (#51)
```

Comment:
I don't think we need to include this, since it was just a change to our testing